server.js

```diff
@@ -1,9 +1,13 @@
 // Should be server.js but since we are not allowed to delete other people's code, a new file.
```

Comment:
Could you remove this comment? It does not serve any purpose.

Reply:
Done @ArnasDickus